src/revwalk.h

```diff
@@ -50,4 +50,22 @@ struct git_revwalk {
 
 git_commit_list_node *git_revwalk__commit_lookup(git_revwalk *walk, const git_oid *oid);
 
+typedef struct {
+	int uninteresting;
+	int from_glob;
+} git_revwalk__schedule_options;
+
+#define GIT_REVWALK__SCHEDULE_OPTIONS_INIT { 0 }
+
+int git_revwalk__schedule_commit(git_revwalk *walk,
+								 const git_oid *oid,
+								 const git_revwalk__schedule_options *opts);
```

Comment:
Indentation of these functions looks a bit funny on GitHub

Reply:
Fixed.